packages/react-microlink/src/index.js

```diff
@@ -48,27 +15,18 @@ export default class extends Component<CardProps, State> {
       this.setState({ loading: true }, () =>
         fetch(url)
           .then(res => res.json())
-          .then((res: Object) => {
-            const {status = '', data}: {status: string, data: Object} = res
-            if (status === 'success') {
-              const { title, description, url, image }: {
-                title?: string,
-                description?: string,
-                url?: string,
-                image?: string | Object
-              } = data
-
-              if (image) {
-                if (typeof image === 'object') {
-                  const {color, background_color: backgroundColor}: {color: string, background_color: string} = image
-                  this.setState({color, backgroundColor})
-                }
-                const imagePath = getUrlPath(image)
-                this.setState({image: imagePath})
-              }
-
-              this.setState({ title, description, url, loading: false })
-            }
+          .then(({status, data}) => {
```

Comment:
How simple looks this code without typechecking around that

Reply:
Yeah it's stripped down the codebase quite a lot, nice

However the type-checking (with a _configured_ IDE etc.) is nice to have. Maybe you're right on it being overkill for _this_ project, but I'd recommend you check Flow out on your next React project 😉